src/engraving/rw/read114/read114.cpp

```diff
@@ -1359,6 +1359,14 @@ static void readTextLine114(XmlReader& e, ReadContext& ctx, TextLine* textLine)
 
 static void readPedal114(XmlReader& e, ReadContext& ctx, Pedal* pedal)
 {
+    bool beginTextTag = false;
+    bool continueTextTag = false;
+    bool endTextTag = false;
+
+    pedal->setBeginText(String());
+    pedal->setContinueText(String());
+    pedal->setEndText(String());
```

Comment:
I realised there's still one issue: if `symbol != pedal->propertyDefault(Pid::BEGIN_TEXT).value<String>()` is false a few lines below, then the begin text will not be set there, so it will have been set to `String()` here at the top of the function.